### PR TITLE
delete color_object_verification and unite it to object_verification

### PR DESCRIPTION
**Pull request by iory**
color_object_verificationをobject_verificationに統合しました

### CODE REVIEW

File: jsk_2014_picking_challenge/scripts/object_verification.py [hunk 1]
```diff
@@ -33,24 +38,31 @@ def _cb_bof(self, msg):
...
-        proba = [(c, objects_proba[c]) for c in candidates]
+        proba = [(c, bof_objects_proba[c] + cfeature_objects_proba[c]) for c in candidates]
+
         matched = sorted(proba, key=lambda x: x[1])[-1][0]
         # compose msg
         msg = ObjectRecognition()
         msg.header.stamp = stamp
         msg.matched = matched
-        msg.probability = objects_proba[matched]
+        msg.probability = max([prob for c, prob in proba])
```
Comment (wkentaro) [inline jsk_2014_picking_challenge/scripts/object_verification.py]: why did change this?

File: jsk_2014_picking_challenge/scripts/object_verification.py [hunk 2]
```diff
@@ -33,24 +38,31 @@ def _cb_bof(self, msg):
...
         msg.candidates = candidates
-        msg.probabilities = [objects_proba[c] for c in candidates]
+        msg.probabilities = np.array([prob for c, prob in proba])
```
Comment (wkentaro) [inline jsk_2014_picking_challenge/scripts/object_verification.py]: Please use candidates as the keys.

**Comment by wkentaro** [comment]
ping @iory

**Comment by wkentaro** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: jsk_2014_picking_challenge/scripts/object_verification.py (hunk 1 above)
Comment (iory) [inline jsk_2014_picking_challenge/scripts/object_verification.py]: I think that in this code objects_proba means sum of probabilities of bof and color_features.
so i change it. 
do you have any good idea?

Comment (wkentaro) [inline jsk_2014_picking_challenge/scripts/object_verification.py]: I thought this can be `msg.probability = dict(proba)[matched]` .
but it's okay.

**Comment by wkentaro** [comment]
ping @iory

**Comment by wkentaro** [comment]
ping does not work..

**Comment by wkentaro** [comment]
I sent PR: https://github.com/iory/2014-semi/pull/2

**Comment by wkentaro** [comment]
マージしてリベースしてください。 @iory

**Comment by wkentaro** [comment]
リベースしてください。
@iory

**Comment by wkentaro** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by wkentaro** [comment]
See #770

**Pull request closed without merging**